src/main/java/core/GridDriver.java

```diff
@@ -0,0 +1,62 @@
+package core;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.remote.CapabilityType;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.openqa.selenium.remote.RemoteWebDriver;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+public class GridDriver {
+
+    public GridDriver() {
+
+    }
+
+        public static WebDriver driver;
+
+        public static WebDriver getDriver(String browser) {
```

Comment:
As far as this comment is out of WebDriver training, it will be optional:
Avoid methods with more than 30 lines of code. I recommend to create a couple of readable private methods.